docs/UserGuide.md

```diff
@@ -126,25 +134,43 @@ Format: `find KEYWORD [MORE_KEYWORDS]`
 Examples:
 * `find John` returns `john` and `John Doe`
 * `find alex david` returns `Alex Yeoh`, `David Li`<br>
-  ![result for 'find alex david'](images/findAlexDavidResult.png)
+  <!--- ![result for 'find alex david'](images/findAlexDavidResult.png) --->
+
+### Filtering residents by field : `filter`
+
+Shows a list of residents whose fields match the input keywords.
+
+Format: `filter [/specifier] KEY:VALUE [ADDITIONAL_KEYS:ADDITIONAL VALUES]`
```

Comment:
Maybe change format to `filter [/SPECIFIER] KEY/VALUE [ADDITIONAL_KEYS/ADDITIONAL_VALUES]`.